core/chains/evm/config/config_test.go

```diff
@@ -369,109 +311,16 @@ func TestChainScopedConfig_Profiles(t *testing.T) {
 }
 
-func Test_chainScopedConfig_Validate(t *testing.T) {
-	configWithChains := func(t *testing.T, id int64, chains ...*toml.Chain) config.AppConfig {
-		return configtest.NewGeneralConfig(t, func(c *chainlink.Config, s *chainlink.Secrets) {
-			s.Database.URL = models.MustSecretURL("postgresql://doesnotexist:justtopassvalidationtests@localhost:5432/chainlink_na_test")
-			chainID := ubig.NewI(id)
-			c.EVM[0] = &toml.EVMConfig{ChainID: chainID, Enabled: ptr(true), Chain: toml.Defaults(chainID, chains...),
-				Nodes: toml.EVMNodes{{
-					Name:    ptr("fake"),
-					WSURL:   configurl.MustParseURL("wss://foo.test/ws"),
-					HTTPURL: configurl.MustParseURL("http://foo.test"),
-				}}}
-		})
-	}
-
-	// Validate built-in
-	for _, id := range toml.DefaultIDs {
-		id := id
-		t.Run(fmt.Sprintf("chainID-%s", id), func(t *testing.T) {
-			cfg := configWithChains(t, id.Int64())
-			assert.NoError(t, cfg.Validate())
-		})
-	}
-
-	// Invalid Cases:
-
-	t.Run("arbitrum-estimator", func(t *testing.T) {
-		t.Run("custom", func(t *testing.T) {
-			cfg := configWithChains(t, 0, &toml.Chain{
-				ChainType: ptr(string(commonconfig.ChainArbitrum)),
-				GasEstimator: toml.GasEstimator{
-					Mode: ptr("BlockHistory"),
-				},
-			})
-			assert.NoError(t, cfg.Validate())
-		})
-		t.Run("mainnet", func(t *testing.T) {
-			cfg := configWithChains(t, 42161, &toml.Chain{
-				GasEstimator: toml.GasEstimator{
-					Mode: ptr("BlockHistory"),
-					BlockHistory: toml.BlockHistoryEstimator{
-						BlockHistorySize: ptr[uint16](1),
-					},
-				},
-			})
-			assert.NoError(t, cfg.Validate())
-		})
-		t.Run("testnet", func(t *testing.T) {
-			cfg := configWithChains(t, 421611, &toml.Chain{
-				GasEstimator: toml.GasEstimator{
-					Mode: ptr("SuggestedPrice"),
-				},
-			})
-			assert.NoError(t, cfg.Validate())
-		})
-	})
-
-	t.Run("optimism-estimator", func(t *testing.T) {
-		t.Run("custom", func(t *testing.T) {
-			cfg := configWithChains(t, 0, &toml.Chain{
-				ChainType: ptr(string(commonconfig.ChainOptimismBedrock)),
-				GasEstimator: toml.GasEstimator{
-					Mode: ptr("BlockHistory"),
-				},
-			})
-			assert.NoError(t, cfg.Validate())
-		})
-		t.Run("mainnet", func(t *testing.T) {
-			cfg := configWithChains(t, 10, &toml.Chain{
-				GasEstimator: toml.GasEstimator{
-					Mode: ptr("FixedPrice"),
-				},
-			})
-			assert.NoError(t, cfg.Validate())
-		})
-		t.Run("testnet", func(t *testing.T) {
-			cfg := configWithChains(t, 69, &toml.Chain{
-				GasEstimator: toml.GasEstimator{
-					Mode: ptr("FixedPrice"),
-				},
-			})
-			assert.NoError(t, cfg.Validate())
-		})
-	})
-}
-
 func TestNodePoolConfig(t *testing.T) {
-	gcfg := configtest.NewGeneralConfig(t, func(c *chainlink.Config, s *chainlink.Secrets) {
-		id := ubig.New(big.NewInt(rand.Int63()))
-		c.EVM[0] = &toml.EVMConfig{
-			ChainID: id,
-			Chain:   toml.Defaults(id, &toml.Chain{}),
-		}
-	})
-	cfg := evmtest.NewChainScopedConfig(t, gcfg)
+	cfg := testutils.NewTestChainScopedConfig(t, nil)
```

Comment:
Why did we ditch the random chain id? That may not have been the best approach, but without anything specified, does this give us simulated defaults now? I don't think that is what was intended.

Reply:
I removed random because if it hits an actual chain id by accident, the default values will change and won't be equal. Here we assume we will always have the fallback values.

Reply:
But does it use the _simulated_ chain id values by default? Those are different than the fallback values.

Reply:
Ok I wasn't aware of the `Simulated.toml` file. Then we should probably use an unassigned fixed value.

Reply:
Fixed chain IDs used to be problematic because they lead to database conflicts which manifest as blocked transactions with txdb. If that is still a problem, then we may need to find a wide range to select from randomly, or pick until we get one that doesn't exist in the default set :shrug:

Reply:
Yeah, I understand the problem but for this specific use case we don't have any db interactions so it seems ok to use a fixed unassigned value. I don't expect the config tests to have an impact on latency.